newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -283,3 +283,18 @@ export const onRenderNewProjectSaveAsLocationChooser = ({
     />
   );
 };
+
+export const isTryingToSaveInForbiddenPath = (
+  saveAsLocation: SaveAsLocation
+): boolean => {
+  const fileIdentifier = saveAsLocation.fileIdentifier;
+  if (!fileIdentifier) return false;
+  // If the user is saving locally and chose the same location as where the
+  // executable is running, prevent this, as it will be deleted when the app is updated.
+  const exePath = remote.app.getPath('exe');
+  if (!exePath) return false; // This should not happen, but let's be safe.
+  const gdevelopDirectory = path.dirname(exePath);
+  const saveAsLocationPath = fileIdentifier;
+  console.log(gdevelopDirectory, saveAsLocationPath);
```

Comment:
To remove

Reply:
yes, I was still testing with logs to ensure the logic makes sense, I'll remove this